src/software/ai/hl/stp/tactic/tactic_update_visitor.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include "software/ai/hl/stp/tactic/tactic_visitor.h"
+#include "software/ai/hl/stp/tactic/all_tactics.h"
+
+class TacticUpdateVisitor : public TacticVisitor
```

Comment:
@MathewMacDougall - thoughts on naming for this? `TacticWorldParamsUpdateVisitor` is technically more specific but is a _bit_ more wordy.

Reply:
I'd be fine with that.

Reply:
If we need to update additional members other than World in the future (looks highly unlikely), is the plan to create a new visitor for each one?

Reply:
If they were a different high level construct that didn't fall under the world, then I'd expect yes